Miscellaneous revisions

aaronw22 opened the pull request:
## Describe your changes


## Do the following before requesting a review

- [ ] I have written tests covering functions I have added or changed.
- [ ] I have run tests and they all pass.
- [ ] I have ensured all new functions will show up in the `_pkgdown.yml` file.
- [ ] I have updated `DESCRIPTION` with any new package dependencies.
- [ ] I have updated `NEWS.md` with a brief description of my changes.
- [ ] I have updated the package documentation with `devtools::document()`.
- [ ] I have updated the package website with `pkgdown::build_site_github_pages()`.

JackBuckley reviewed (commented):
I realised that a few of the changes to the save functions clash with edits I mad to fix a couple of small bugs. 
I've just gone ahead and added your changes to save_e61, save_single, save_multi etc. to my branch and merged them.
So if you can drop those changes and merge everything else that would be ideal.

aaronw22 commented:
> I realised that a few of the changes to the save functions clash with edits I mad to fix a couple of small bugs. I've just gone ahead and added your changes to save_e61, save_single, save_multi etc. to my branch and merged them. So if you can drop those changes and merge everything else that would be ideal.

I've fixed the merge conflicts and added a few improvements to things as well.

I've added the changes to flipped charts that we were discussing, for both the title and the caption text. Currently the user will have to manually decide on the character width of the caption to fit the graph.
